invenio_cli/commands/requirements.py

```diff
@@ -125,8 +126,9 @@ def check_docker_compose_version(cls, major, minor=-1, patch=-1, exact=False):
         """Check the docker compose version."""
         # Output comes in the form of
         # 'Docker Compose version v2.17.3\n'
+        docker_helper = DockerHelper("", local=False)
```

Comment:
I guess this is not how it's supposed to be, but I can't call `cli_config.get_project_shortname()` as in https://github.com/inveniosoftware/invenio-cli/blob/5f89470b8015b716a4b63f3163214595ff7f74ee/invenio_cli/commands/services.py#L30 in this context.